webpack.config.js

```diff
@@ -1,16 +1,20 @@
 /* eslint-disable flowtype/require-valid-file-annotation */
 /* eslint-disable import/no-nodejs-modules*/
 const path = require('path');
-const glob = require('glob');
 const webpack = require('webpack');
 const CleanObsoleteChunksPlugin = require('webpack-clean-obsolete-chunks');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const {styles} = require('@ckeditor/ckeditor5-dev-utils'); // eslint-disable-line import/no-extraneous-dependencies
 
-const entries = glob.sync(
-    path.resolve(__dirname, 'src/Sulu/Bundle/*/Resources/js/index.js') // eslint-disable-line no-undef
-);
+const entries = [];
+
+entries.unshift('sulu-admin-bundle');
+entries.unshift('sulu-content-bundle');
+entries.unshift('sulu-media-bundle');
+entries.unshift('sulu-preview-bundle');
+entries.unshift('sulu-security-bundle');
```

Comment:
I like the change because these files are contained within our control anyway. The glob would only make sense if files from 3rd party bundles would be included as well. Maybe we should even change the `glob` to do so? Already thought about renaming `index.js` to `sulu-index.js`, so that it could be added to any bundle and it would be automatically discovered, making it easier to use.

But i don't see how this solves problems when moving files to `assets/admin`...

Reply:
couldn't find any problems with cache as npm symlinks and yarn seems to copy the folder always.